Sources/AsyncKit/ConnectionPool/EventLoopConnectionPool.swift

```diff
@@ -81,10 +72,10 @@ public final class EventLoopConnectionPool<Source> where Source: ConnectionPoolS
         source: Source,
         maxConnections: Int,
         requestTimeout: TimeAmount = .seconds(10),
+        pruneInterval: TimeAmount = .seconds(60),
+        maxIdleTimeBeforePruning: TimeAmount = .seconds(120),
         logger: Logger = .init(label: "codes.vapor.pool"),
-        on eventLoop: EventLoop,
-        pruneInterval: TimeInterval = 60,
-        maxIdleTimeBeforePrunning: TimeInterval = 120
+        on eventLoop: EventLoop
```

Comment:
@Joannis initially I added `maxIdleTimeBeforePruning` and then `pruneInterval` but I realized that we don't really need the `pruneInterval`, we could just put `maxIdleTimeBeforePruning / 2`. This will simplify the result. The reason is that the `pruneInterval` it's an implementation detail.